boards/modalai/fc-v1/src/board_config.h

```diff
@@ -263,6 +263,8 @@
 
 #define BOARD_NUM_IO_TIMERS 5
 
+// J4 / TELEM3 / UART2
+#define MODALAI_ESC_DEFAULT_PORT                  "/dev/ttyS1"
```

Comment:
We should probably replace this with configurable serial parameters (but it's fine like this for now).

Reply:
agreed, will add to my todos